### PR TITLE
[docs] add more faq and FE debugging method

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

1. Add more faq and FE debugging method.
2. Add security document.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/community/security.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 安全
```
Comment (jackwener) [inline docs/en/community/security.md:27]: there is chinese

File: docs/en/faq/faq.md [hunk 2]
```diff
@@ -316,4 +316,27 @@ One situation is OVERCROWDED, which means that a large amount of unsent data at
...
+```
+1, "abc"
+1, "def"
+```
+
+Then the result of replica 1 may be '1, "ABC', while the result of replica 2 may be '1," def'. This leads to inconsistent query results.
+
+To ensure the unique data order between different replicas, refer to the  [Sequence Column](../administrator-guide/load-data/sequence-column-manual.md) function.
+
+### Q23. ` recoveryTracker should overlap or follow on disk last VLSN of 4,422,880 recoveryFirst= 4,422,882 UNEXPECTED_STATE_FATAL`
```
Comment (jackwener) [inline docs/en/faq/faq.md:338]: redundant space.

File: docs/en/community/security.md (hunk 1 above)
Comment (jackwener) [inline docs/en/community/security.md:27]:
```suggestion
# Security
```

File: docs/en/faq/faq.md (hunk 2 above)
Comment (jackwener) [inline docs/en/faq/faq.md:338]:
```suggestion
### Q23. `recoveryTracker should overlap or follow on disk last VLSN of 4,422,880 recoveryFirst= 4,422,882 UNEXPECTED_STATE_FATAL`
```

Comment (morningman) [reply docs/en/faq/faq.md]: All fixed

**Review by yangzhg — DISMISSED** [review]
LGTM

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**